subprojects/zap-clientapi/src/main/java/org/zaproxy/clientapi/core/ClientApi.java

```diff
@@ -361,13 +361,16 @@ private static URL buildZapRequestUrl(
         sb.append('/');
         if (params != null) {
             sb.append('?');
+	    String amp = "";

```

Comment:
Another option would be something like:
String sep = "?"; (and remove the sb.append('?');)
...
sb.append(sep);
...
sep = "&";

(if you see what I mean).